specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/examples/cloud-events-schema/chat_typing_indicator_received_in_thread.json

```diff
@@ -0,0 +1,19 @@
+{
+  "source": "/subscriptions/{subscription-id}/resourceGroups/{group-name}/providers/Microsoft.Communication/communicationServices/{communication-services-resource-name}",
+  "subject": "thread/{thread-id}/sender/{rawId}/recipient/{rawId}",
```

Comment:
Does this make sense to customer to include all these {} fields? Shouldn't this be plain text?